src/index.js

```diff
@@ -0,0 +1,39 @@
+/* eslint no-unused-vars: "off" */
+/* eslint no-restricted-globals: "off" */
+import css from './style.css';
+import createScore from './createScore.js';
+import loadStored from './loadStored.js';
+import postToAPI from './postToAPI.js';
+import emptyBoard from './emptyBoard.js';
+
+const form = document.querySelector('form');
+const formName = document.querySelector('input#name');
+const formScore = document.querySelector('input#score');
+const refresh = document.querySelector('button.btn');
+
+async function getStorage() {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3IGjFzzBEY0MX1krdUtm/scores/').then((response) => response.json()).then((json) => {
+    const list = json.result;
+    loadStored(list);
+  });
+}
```

Comment:
**Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.